decomposition.tex

```diff
@@ -137,21 +137,21 @@
     // addressability
     { &t } -> Same<T*>;
```

Comment:
> Oh wait. I don't think you meant to change this here. This is the Movable concept from the Palo Alto report. But the larger issue remains.

Yeah, wrong place but right idea. I'll specify all four variants explicitly in `Destructible` and look for similar occurrences. Hopefully something better will come to us later.